csrankings-f.csv

```diff
@@ -116,6 +116,7 @@ Fang Liu 0002,Hunan University,http://design.hnu.edu.cn/info/1023/5787.htm,PXo2a
 Fang Song 0001,Portland State University,https://fangsong.info,A6C3geAAAAAJ
 Fang Zhao 0003,BUPT,https://scs.bupt.edu.cn/info/1277/2497.htm,NOSCHOLARPAGE
 Fangchu Yang,BUPT,https://int.bupt.edu.cn/content/content.php?p=6_16_94,NOSCHOLARPAGE
+Fanghui Liu 0001,https://www.lfhsgre.org/,AKxBgssAAAAJ

```

Comment:
No institution name.

Reply:
thanks for your information. I have alread added the affiliation.